src/shared/components/title-fades/title-fades.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-title-fades',
+  templateUrl: './title-fades.component.html',
+  styleUrls: ['./title-fades.component.scss'],
+})
+export class TitleFadesComponent implements OnInit {
+  constructor() {}
+
+  @Input() color: string = '';
```

Comment:
what will happen if we have an input of "black" string?

Reply:
I modify it to render red title with problem description